### PR TITLE
Link to album when any and show graphics on the front. 

### PR DESCRIPTION
**Pull request by dhakelila**
https://basecamp.com/1756858/projects/12531475/todos/290504032

· Showing a link to visit the album a pic or a graph belongs to in the breadcrumbs and below the download button. Maybe we can choose a better text. 

· Showing the graphs if any in the front.

### CODE REVIEW

File: app/views/media_library/show.html.erb [hunk 1]
```diff
@@ -33,11 +33,18 @@
     <div class="row">
       <div class="small-12 columns">
         <div class="title-block">
-          <% breadcrumbs_links = [
+          <% breadcrumbs_links = @media_content.album_id.present? ? [
```
Comment (simaob) [inline app/views/media_library/show.html.erb]: In ruby I prefer to have this if like this:

```
breadcrumbs_links = if @media_content.album_id.present? 
                      [something]
                   else
                       [something_else]
                   end
```

File: app/views/media_library/show.html.erb [hunk 2]
```diff
@@ -33,11 +33,18 @@
...
             {text: 'Media library', link: media_path},
+            {text: 'Album', link: medium_path(@media_content.album_id)},
```
Comment (simaob) [inline app/views/media_library/show.html.erb]: Could you try with displaying the album name. You can use truncate to make it smaller

`truncate(@media_content.album.title, length: 20)`

Comment (dhakelila) [inline app/views/media_library/show.html.erb]: Sure! I can. 💪

**Comment by dhakelila** [comment]
-Showing album title.
-Better syntaxis for if statement and move that bar with the others.

**Pull request merged**